src/resources/parser/glb-parser.js

```diff
@@ -1356,14 +1355,23 @@ const createAnimation = function (gltfAnimation, animationIndex, gltfAccessors,
         'weights': 'weights'
     };
 
+    const constructNodePath = (node) => {
+        const path = [];
+        while (node) {
+            path.splice(0, 0, node.name);
```

Comment:
why not using unshift? That's easier to read

Reply:
Ahh yes I did a quick search when I wrote this code and missed unshift. Thanks!